WooCommerce/Classes/Tools/Zendesk/ZendeskManager.swift

```diff
@@ -597,11 +597,11 @@ final class ZendeskManager: NSObject, ZendeskManagerProtocol {
 ///
 extension ZendeskManager {
     func formID() -> Int64 {
-        createRequest(supportSourceTag: nil).ticketFormID?.int64Value ?? .zero
```

Comment:
These were changed because obtaining the form ID via the `createRequest` method was not very performant due to how `SwiftUI` constantly queries for equality.

In any case, I expect to refactor those accessors on https://github.com/woocommerce/woocommerce-ios/issues/8795

Reply:
Good catch!

I found the `formID()` and `wcPayFormID` names confusing based on how they're used, but it's not a big deal if you plan to refactor them.